packages/landingpage-new/pages/[lang]/library/components/index.module.scss

```diff
@@ -3,3 +3,19 @@
   justify-content: center;
   align-items: center;
 }
+
+.openExternallyButton {
+  position: fixed;
+  bottom: 22px;
```

Comment:
I suggest moving the button to a positon of `top: 60px` so that the button doesn't have a starting position which overlaps the storybook button 'open fullscreen'.

current:
<img width="95" alt="image" src="https://github.com/inovex/elements/assets/103122411/4f3ae526-dc66-4a1f-957f-5e02ee3d6f4d">

suggestion:
<img width="103" alt="image" src="https://github.com/inovex/elements/assets/103122411/313388ec-fc26-4bba-86c1-d1c7bc644405">